### DISCUSSION TITLE
Application Approved but didn't receive Benefits

### DISCUSSION THREAD

**Discussion by scoxfield** [community/community#143917] [GitHub Education]
### Hi everyone,

I recently applied for the GitHub Student Pack and was approved, but it has been 4-5 days and I have not yet received access to the benefits.

I am planning to participate in a hackathon tomorrow, and the pack would be incredibly helpful for that. Could you kindly assist me in gaining access to the features?

Thank you for your time and support. I look forward to your response.

**Comment by huddmax** [comment]
Hi! 👋 

Congrats on being approved for the GitHub Student Pack! 🎉 Usually, it can take a few days for the benefits to fully activate in your account. However, if you’re in a time crunch for the hackathon, here’s what you can do:

1. **Log out and log back in**: Sometimes, refreshing your session can help activate the benefits.
2. **Check Your Email**: You might receive an email confirmation with further steps or links to access specific benefits. Make sure to follow any instructions there.
3. **Contact GitHub Education Support**: Since you have an upcoming event, reach out to [GitHub Education Support](https://support.github.com/contact/education) and explain the situation. They may be able to expedite your access.

Good luck at the hackathon! 🚀

**Comment by davevad93** [comment]
Hi @scoxfield ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @scoxfield was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
